runtime/tests/annotationtests/annhelp_tests.c

```diff
@@ -82,3 +82,58 @@ Java_org_openj9_test_annotation_ContainsRuntimeAnnotationTest_containsRuntimeAnn
 
 	return result;
 }
+
+/**
+ * Checks if a method contains the runtime annotation specified.
+ *
+ * @param env The JNI environment.
+ * @param method The reflect method to query
+ * @param annotationNameString The name of the annotation to check for.
+ * @return JNI_TRUE if the annotation is found, JNI_FALSE otherwise.
+ */
+jboolean JNICALL
+Java_org_openj9_test_annotation_ContainsRuntimeAnnotationTest_methodContainsRuntimeAnnotation(JNIEnv *env, jclass unused, jobject method, jstring annotationNameString)
+{
+	jboolean result = JNI_FALSE;
+	J9VMThread *currentThread = (J9VMThread *)env;
+	J9JavaVM *vm = currentThread->javaVM;
+	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
+	PORT_ACCESS_FROM_JAVAVM(vm);
+
+	if (NULL == annotationNameString) {
+		vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGNULLPOINTEREXCEPTION, "annotation name is null");
+	} else if (NULL == method) {
+		vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGNULLPOINTEREXCEPTION, "method is null");
+	} else {
+		vmFuncs->internalEnterVMFromJNI(currentThread);
+
+		j9object_t annotationNameObj = J9_JNI_UNWRAP_REFERENCE(annotationNameString);
+		char annotationNameStackBuffer[J9VM_PACKAGE_NAME_BUFFER_LENGTH] = {0};
+		J9UTF8 *annotationNameUTF8 = vmFuncs->copyStringToJ9UTF8WithMemAlloc(
+																			currentThread,
```

Comment:
can you move the indentation a bit to the left here

Reply:
4 space tabs aligned it to the `(`. Reduced tabs by half